p2p/src/net/libp2p/tests/frontend.rs

```diff
@@ -307,9 +291,11 @@ async fn test_connect_with_timeout() {
     let config = Arc::new(common::chain::config::create_mainnet());
     let (mut service, _, _) = Libp2pService::start(
         test_utils::make_libp2p_addr(),
-        &[],
         config,
-        Duration::from_secs(2),
+        Arc::new(config::P2pConfig {
+            outbound_connection_timeout: 2,
+            ..Default::default()
```

Comment:
No `..` please.

Reply:
these are tests

Reply:
I know they're tests... but still not very happy about them. If you insist on keeping them like this, I'm fine with that. But I think the right thing to do is not use `..` so that you even check the tests when a change is done.